Makefile

```diff
@@ -1,26 +1,40 @@
 DC := $(shell which docker-compose)
+HOSTUSER := $(shell id -u):$(shell id -g)
 
 default:
-	@echo "You need to specify a subcommand."
+	@echo "You need to specify a subcommand. Type 'make help' for help."
 	@exit 1
 
 help:
-	@echo "build         - build docker containers for dev"
-	@echo "shell         - open a shell in the base container"
-	@echo "test          - run tests"
+	@echo "build        - install Python libs and build Docker containers"
+	@echo "test         - run tests"
+	@echo "testshell    - open a shell in the test container"
+	@echo "clean        - remove build files"
 
-# Dev configuration steps
-.docker-build:
-	make build
+.container-test: docker/test/Dockerfile requirements-dev.txt
+	${DC} build test
+	touch .container-test
```

Comment:
I like these sorts of things. This will re-run building the test container if `Dockerfile` or `requirements-dev.txt` change. That's nice.